cpp/common/src/codingstandards/cpp/Scope.qll

```diff
@@ -287,3 +299,18 @@ predicate hasClassScope(Declaration decl) { exists(decl.getDeclaringType()) }
 
 /** Holds if `decl` has block scope. */
 predicate hasBlockScope(Declaration decl) { exists(BlockStmt b | b.getADeclaration() = decl) }
+
+/**
+ * identifiers in nested (named/nonglobal) namespaces are exceptions to hiding due to being able access via fully qualified ids
+ */
+predicate excludedViaNestedNamespaces(UserDeclaration v2, UserDeclaration v1) {
```

Comment:
```suggestion
predicate excludedViaNestedNamespaces(UserDeclaration v1, UserDeclaration v2) {
```
Can we use the same order as `hides`, where `v2` hides `v1`.

Better names would be `outerDecl` and `innerDecl` where `innerDecl` hides the `outerDecl`.